components/Dashboard/FormEdit.tsx

```diff
@@ -123,15 +125,21 @@ const FormEdit: React.FC<FormEditProps> = ({ form, onStepChange }) => {
               <FormControl>
                 <div className="relative mt-4 w-fit">
                   <input
+                    ref={inputref}
                     disabled={form.formState.isSubmitting}
                     className="w-[7.5rem] rounded bg-transparent px-1 py-1 text-xl font-bold outline-dashed outline-1 outline-transparent duration-100 hover:outline-gray-300 focus:outline-gray-300"
-                    {...field}
```

Comment:
Why did we removed ...field here?

Reply:
> Why did we removed ...field here?

Because it was throwing an error saying "'ref' is specified more than once, so this usage will be overwritten. and 
`{...field}` This spread always overwrites this property."

Reply:
@piyushgarg-dev This is the video showing the working of the pencil icon as you requested.

https://github.com/piyushgarg-dev/review-app/assets/131521505/8be532a0-cb13-4214-991f-964d4e286cf7